test/e2e/websocket_test.go

```diff
@@ -83,9 +92,10 @@ func connect(t *testing.T, clients *test.Clients, domain string) (*websocket.Con
 	return conn, waitErr
 }
 
-func validateWebSocketConnection(t *testing.T, clients *test.Clients, names test.ResourceNames) error {
-	var err error
+const message = "Hello, websocket"
 
+func validateWebSocketConnection(t *testing.T, clients *test.Clients, names test.ResourceNames, prefix bool) error {
```

Comment:
This whole prefix/suffix bit seems a bit strange. Why not just take in the message rather than a boolean?

Reply:
I need revisions to return different strings. 
We can write a whole new WS server for that (or use a different path), but this seems the easiest way forward and is quite straightforward.
Now that I am really proud of it :)